scripts/rundnsmasq

```diff
@@ -13,7 +13,11 @@ export DNS_PORT=${DNS_PORT:-0}
 
 wait_for_interface_or_ip
 if [[ "${DNS_IP:-}" == "provisioning" ]]; then
-    export DNS_IP="$IRONIC_URL_HOST"
+    if [[ "$IPV" == "4" ]]; then
```

Comment:
Nit: ideally this fix should have been separate to be able to backport it.

Reply:
would you like me to pull it out and open another PR?

Reply:
Only if you have spare cycles